content/en/smart-contracts/wallets/foxwallet/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "FoxWallet"
+description: "This page explains what FoxWallet is, and how to integrate it into the Filecoin network."
+lead: "FoxWallet fully supports the Filecoin ecosystem from scratch. It has integrated the Filecoin EVM-runtime smart contract underhood, thus providing users with great experience transferring FIL and from and to f4 address without manual setup."
```

Comment:
Phrasing is difficult to parse here, suggested edit

```suggestion
lead: "FoxWallet is a decentralized, multi-chain wallet with full support for Filecoin. It integrates with the Filecoin Ethereum Virtual Machine (FEVM), allowing users to easily transfer FIL to and from `f4` address without manual setup. This guide explains how to transfer FIL using Foxwallet."
```

Reply:
After rewriting this, I'm actually not sure I rewrote it correctly. It's not clear what the intended goal of this guide is to me